### PR TITLE
Added delete column to un-billed expenses column

### LINKED ISSUE #259: add the ability to delete a row from unbilled expenses
viewing any unbilled expenses, ex: /expense/unbilled/25, add a column to allow deleting the row.  there should be a confirmation required, but it doesn't have to be a fancy one.  Delete the row on the db on confirm and then it should remove itself from the table too.

needs to be done after #258 is merged

### PR DESCRIPTION
**Pull request by wongband**
resolve #259

### CODE REVIEW

**Review by srkirkland — APPROVED** [review]

File: Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx [hunk 1]
```diff
@@ -50,9 +74,34 @@ export const UnbilledExpensesContainer = () => {
...
+            <Button
+              color="success"
+              onClick={() =>
+                selectedExpense !== null
+                  ? confirmModal(selectedExpense.id)
+                  : null
+              }
+            >
+              Confirm
+            </Button>{" "}
```
Comment (jSylvestre) [inline Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx:93]: What is the {" "} for?

**Review by jSylvestre — APPROVED** [review]

File: Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx (hunk 1 above)
Comment (wongband) [reply Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx:93]: It creates space between the two buttons

Comment (jSylvestre) [reply Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx:93]: Hmm, I removed it and didn't notice any difference.
Probably better to use a class for this kind of formatting. @cydoval ?

Comment (srkirkland) [reply Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx:93]: Empty string is actually pretty common way to do things. But yea there might be something better. But you do see it this way a lot, especially when prettier reformats things.

**Pull request merged**